src/database/schema/ChatSchema.ts

```diff
@@ -355,8 +403,10 @@ class ChatSchema implements BaseDbSchema {
         response?.message?.updated_at,
         JSON.stringify(response),
         response?.message?.id,
+        JSON.stringify(response?.message?.attachments),
         this.id
       ];
+      console.warn('updateReplacement', JSON.stringify(updateReplacement));
 
       await db.executeSql(updateQuery, updateReplacement);
     } catch (e) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [397-408]

In the SQL query, you're updating the `id` field. Generally, IDs should be immutable once set. If you need a temporary ID for unsent messages, consider using a different field for that purpose.

```diff
-         SET status = ?, created_at = ?, updated_at = ?, raw_json = ?, id = ?, attachment_json = ?
+         SET status = ?, created_at = ?, updated_at = ?, raw_json = ?, attachment_json = ?
```
And remove `response?.message?.id,` from `updateReplacement`.

If you still need to update the `id`, ensure that it won't cause issues with foreign keys or other references to this ID.


<!-- This is an auto-generated comment by OSS CodeRabbit -->